examples/arithmetics/src/language-server/generated/meta-data.ts

```diff
@@ -0,0 +1,9 @@
+/******************************************************************************
+ * This file was generated by langium-cli 0.1.0.
+ * DO NOT EDIT MANUALLY!
+ ******************************************************************************/
+
+export class ArithmeticsLanguageMetaData {
```

Comment:
In this case I'd rather use a plain object than a class.

Reply:
I see, that makes sense. Should the object this be generated into a `meta-data.ts` file, or should it be build in the generated injection module? like this:

```ts
LanguageMetaData: () => ({
    languageId: 'id',
    extensions: ['.id']
})
```

Reply:
Exporting a `const` for it could be useful so you can access it directly in your language implementation. But it could be in the `generated/module.ts` instead of a new file to keep the number of generated files low.

Reply:
Alright, yeah I agree with that.